run_configs/lfric/lfric_common.py

```diff
@@ -21,79 +23,113 @@ def check_available(self):
         return True
 
 
-# todo: is this part of psyclone? if so, put  it in the psyclone step module?
```

Comment:
Just to be clear (and I think you understand this already) "Configurator" is its own tool unconnected to PSyclone.

Reply:
Yep, that's why I removed the comment :)